ebos/eclwriter.hh

```diff
@@ -372,9 +376,44 @@ private:
 
     Opm::NNC exportNncStructure_() const
     {
-        Opm::NNC nnc = eclState().getInputNNC();
-        int nx = eclState().getInputGrid().getNX();
-        int ny = eclState().getInputGrid().getNY();
+        std::size_t nx         = eclState().getInputGrid().getNX();
```

Comment:
no ascii art for aligning lines please. (because this makes it basically impossible to search-and-replace variable names without breaking the formatting.)

Reply:
I doubt this statement. I recall having done this with emacs search-and-replace-regex. 
On the other hand I do not care and will change it just to make you happy.

Reply:
I constantly have problems with this and if the author does not notice this, it results in botched-up formatting. I'm pretty sure that you can find quite a few instances of this in the OPM and DUNE code bases...